modules/imgcodecs/test/test_tiff.cpp

```diff
@@ -333,6 +333,31 @@ TEST(Imgcodecs_Tiff, decode_black_and_write_image_pr12989_default)
     EXPECT_EQ(CV_8UC3, img.type()) << cv::typeToString(img.type());
 }
 
+TEST(Imgcodecs_Tiff, decode_black_and_write_image_pr12989)
```

Comment:
Please rename new tests, they are duplicated.
Did you run these tests locally?

Reply:
Sorry for that, I have updated the name and ran these tests locally. Thanks!